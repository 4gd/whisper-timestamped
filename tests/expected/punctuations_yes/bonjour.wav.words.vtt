WEBVTT

00:00.220 --> 00:00.980
Bonjour!

