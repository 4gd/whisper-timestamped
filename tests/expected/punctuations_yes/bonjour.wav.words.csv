Bonjour!,0.16,0.78
